include/glow/Runtime/RuntimeTypes.h

```diff
@@ -48,6 +48,17 @@ using ResultCBTy = std::function<void(
 struct DeviceInfo {
   /// Available memory on device in bytes.
   size_t availableMemory;
+  /// Available SRAM capacity in bytes.
+  size_t sramCapacity;
+  /// Peak compute on device in ops/second. Assumes all ops are in int8.
+  /// TODO: distinguish between data types with different peak flops.
+  float peakCompute;
```

Comment:
Ideally yes. In general, it would be good to get the characteristics of the architecture from querying some vendor API. This current design is the first step towards that.